docs/src/main/asciidoc/security-authentication-mechanisms-concept.adoc

```diff
@@ -198,6 +198,7 @@ For more information about OIDC authentication and authorization methods you can
 |Bearer token authentication mechanism |xref:security-oidc-bearer-token-authentication-concept.adoc[OIDC Bearer authentication]
 |Authorization code flow authentication mechanism |xref:security-oidc-code-flow-authentication-concept.adoc[OpenID Connect (OIDC) authorization code flow mechanism]
 |Multiple tenants that can support bearer token or authorization code flow mechanisms |xref:security-openid-connect-multitenancy.adoc[Using OpenID Connect (OIDC) multi-tenancy]
+|Well-Known OpenID Connect Providers |xref:security-openid-connect-providers.adoc[Configuring Well-Known OpenID Connect Providers]
```

Comment:
Use sentence case in the left column cell.

_The linked topic title & xref label should also use sentence case as per Quarkus doc title guidelines, but this is a separate diataxis restructuring task._ 

Minor suggestion. From a findability perspective, you could use an alternative keyword/term in the left column that a customer might also use to find this content. For example:

```suggestion
|Securing Quarkus with commonly used OpenID Connect providers |xref:security-openid-connect-providers.adoc[Configuring Well-Known OpenID Connect Providers]
```